hack/update/github.go

```diff
@@ -43,6 +43,12 @@ type Release struct {
 // GHReleases returns greatest current stable release and greatest latest rc or beta pre-release from GitHub owner/repo repository, and any error occurred.
 // If latest pre-release version is lower than the current stable release, then it will return current stable release for both.
 func GHReleases(ctx context.Context, owner, repo string) (stable, latest, edge Release, err error) {
+	return GHReleasesWithCondition(ctx, owner, repo, nil)
+}
+
+// GHReleasesWithCondition returns greatest current stable release and greatest latest rc or beta pre-release from GitHub owner/repo repository whose version number satisfy the condition given in the parameter, and any error occurred.
+// If latest pre-release version is lower than the current stable release, then it will return current stable release for both.
```

Comment:
```suggestion
// GHReleasesWithCondition returns greatest current stable release and greatest 
// latest rc or beta pre-release from GitHub owner/repo repository whose version
// number satisfy the condition given in the parameter, and any error occurred. 
// If latest pre-release version is lower than the current stable release, then 
// it will return current stable release for both.
```